BuildTools.sln

```diff
@@ -92,6 +92,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Internal.AspNetCore.SiteExt
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGetPackageVerifier.Tests", "test\NuGetPackageVerifier.Tests\NuGetPackageVerifier.Tests.csproj", "{439CC7A3-F6E6-46B8-B6A0-05E22E558FC2}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.BuildTools.CreateLZMA", "src\Microsoft.AspNetCore.BuildTools.CreateLZMA\Microsoft.AspNetCore.BuildTools.CreateLZMA.csproj", "{27679B88-FED9-404A-8A41-02BB31CF3019}"
```

Comment:
Can we call this `Internal.AspNetCore.Xyz` to match other stuff? I want to make it as clear as possible this isn't a product feature.

Reply:
+1. How about Internal.AspNetCore.Tasks.Lzma for now. I'd avoid naming it CreateLZMA for now because we may want to add more tasks in the future, such as extracting an LZMA.